to.etc.domui/src/main/java/to/etc/domui/util/exporters/ExcelExportWriter.java

```diff
@@ -321,6 +328,17 @@ static public <V> IExportCellRenderer<ExcelExportWriter<?>, Cell, V> findRendere
 		@Override
 		public void renderCell(ExcelExportWriter<?> exporter, Cell cell, int cellIndex, @Nullable Number value) throws Exception {
 			cell.setCellStyle(exporter.m_numberStyle);
+			if(null != value) {
+				cell.setCellValue(value.longValue());
```

Comment:
This is wrong; if the value is BigInteger it could easily not fit. You can cast the rest to long, but BigInteger needs to be set using doubleValue. Use instanceof to test.